modules/ROOT/pages/index.adoc

```diff
@@ -28,7 +27,7 @@ Additionally, you can interact with {productname} using the XML-RPC API.
 * For more information about [command]``spacecmd``, see xref:reference:spacecmd-intro.adoc[].
 * For more information about other command line tools, see xref:reference:command-line-tools.adoc[].
 ifeval::[{suma-content} == true]
-* For more information about the API, see https://documentation.suse.com/external-tree/en-us/suma/4.2/pdf/susemanager_api_doc_color_en.pdf.
+* For more information about the API, see link:https://documentation.suse.com/suma/4.2/pdf/4.2_pdf_susemanager_api_doc_color_en.pdf[].
```

Comment:
usually, we do not need this link decoration.  maybe, it is different now, or in this special case.